docker/fastq_screen/1.0.0-alpha/fastq_screen.conf

```diff
@@ -0,0 +1,127 @@
+# This is a configuration file for fastq_screen
```

Comment:
Can we clean up this file by removing all of the comments?

Reply:
I meant any line that starts with `#` or a blank line should be removed. E.g.

```
##############
## Databases #
##############
#########
## Human - sequences available from
## ftp://ftp.ensembl.org/pub/current/fasta/homo_sapiens/dna/
DATABASE	Human	/tmp/fastq-screen-db/Human/Homo_sapiens.GRCh38

#########
## Mouse - sequence available from
## ftp://ftp.ensembl.org/pub/current/fasta/mus_musculus/dna/
DATABASE	Mouse	/tmp/fastq-screen-db/Mouse/Mus_musculus.GRCm38
```

would become

```
DATABASE	Human	/tmp/fastq-screen-db/Human/Homo_sapiens.GRCh38
DATABASE	Mouse	/tmp/fastq-screen-db/Mouse/Mus_musculus.GRCm38
```

You might keep one comment at the top saying something like:

```
## Data available from the following sources:
##
## ftp://ftp.ensembl.org/pub/current/fasta/homo_sapiens/dna/
## ftp://ftp.ensembl.org/pub/current/fasta/mus_musculus/dna/
## ...etc...
```